### PR TITLE
LPS-198393 Add currentDate in expression builder and notification general terms

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://liferay.atlassian.net/browse/LPS-198797

### CODE REVIEW

**Review by seltong — CHANGES_REQUESTED** [review]
I finished my review, please let me know if you have any questions

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java [hunk 1]
```diff
@@ -112,6 +119,24 @@ public static List<Map<String, Object>> getCodeEditorElements(
...
+			"add-day"),
+		ADD_MONTH(
+			"addMonth(field_name, months)",
+			"calculates-the-result-of-adding-or-subtracting-a-specified-" +
+				"number-of-months-from-a-given-date.-To-subtract,-prefix-the-" +
+					"number-of-days-with-a-minus-sign",
+			"add-month"),
+		ADD_YEAR(
+			"addYear(field_name, years)",
+			"calculates-the-result-of	-adding-or-subtracting-a-specified-" +
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java:136]: This tab really wrong, thank you

**Comment by murilostodolni** [comment]
ci:test:sf

**Review by seltong — APPROVED** [review]
LGTM! :tada:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/AddDaysFunction.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * @author Murilo Stodolni
+ */
+public class AddDaysFunction
+	implements DDMExpressionFunction.Function2<Object, Object, Object> {
+
+	public static final String NAME = "addDays";
+
+	@Override
+	public Object apply(Object object1, Object object2) {
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/AddDaysFunction.java:23]: Yes, done!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/AddYearsFunction.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * @author Murilo Stodolni
+ */
+public class AddYearsFunction
+	implements DDMExpressionFunction.Function2<Object, Object, Object> {
+
+	public static final String NAME = "addYears";
+
+	@Override
+	public Object apply(Object object1, Object object2) {
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/AddYearsFunction.java:23]: Done

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java [hunk 4]
```diff
@@ -112,6 +119,24 @@ public static List<Map<String, Object>> getCodeEditorElements(
...
+			"addDays(field_name, parameter)",
+			"calculates-the-result-of-adding-or-subtracting-a-specified-" +
+				"number-of-days-from-a-given-date.-to-subtract,-prefix-the-" +
+					"number-of-days-with-a-minus-sign",
+			"add-days"),
+		ADD_MONTHS(
+			"addMonths(field_name, parameter)",
+			"calculates-the-result-of-adding-or-subtracting-a-specified-" +
+				"number-of-months-from-a-given-date.-to-subtract,-prefix-the-" +
+					"number-of-days-with-a-minus-sign",
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java:132]: Yes, you're right.

**Review by carolmariaabb — APPROVED** [review]

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:stable

**Comment by murilostodolni** [comment]
ci:test:stable

**Comment by andreefarias** [comment]
ci:test:stable

**Comment by murilostodolni** [comment]
ci:close

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**